websites/C/ChatGPT/presence.ts

```diff
@@ -1,20 +1,50 @@
-const presence = new Presence({ clientId: "1102935778570547282" }),
+const presence = new Presence({
+		clientId: "1102935778570547282",
+	}),
+	getStrings = async () => {
+		return presence.getStrings(
+			{
+				aiResponding: "chatgpt.aiResponding",
+				conversationStats: "chatgpt.conversationStats",
+				startNewConversation: "chatgpt.startNewConversation",
+				talkingWithAI: "chatgpt.talkingWithAI",
+				thinkingOfPrompt: "chatgpt.thinkingOfPrompt",
+			},
+			await presence.getSetting<string>("lang").catch(() => "en")
```

Comment:
The language parameter is deprecated. (language is determined automatically based on user/activity settings)
```suggestion
```